### DISCUSSION TITLE
Renovate renovate_child containers on SELinux enabled systems

### DISCUSSION THREAD

**Discussion by xoxys** [renovatebot/renovate#13422] [Request Help]
Hi,
as discussed in https://github.com/renovatebot/renovate/discussions/13027 and https://github.com/renovatebot/docker-renovate#samples this fails on systems with enabled SELinux... The reason is that the renovate_child container tries to bind mount the `/tmp` folder to the child again without `--privileged` or `-v /tmp:/tmp:Z` permission flag. As a result, the access to the /tmp folder is denied on every child container.

Not sure why you use bind mounts in general (it seems to me that there is also no option to overwrite it, `/tmp` is hard-coded) and not a native docker volume instead, which would avoid any SELinux issues by design and also don't expose all repo data world readable  in the `/tmp` directory especially when `/tmp/renovate` is getting created with `0755` which is also pretty bad in my opinion.

**Comment by rarkins** [comment]
You filed this under questions and answers but there's no question mark after any sentence. Do you have questions, or suggestions, or what do you hope to achieve?

**Reply by xoxys** [reply]
Q: This implementation is not secure and broken on SELinux systems. How to handle it? Create an issue? Would you like to fix it? Or is it intended to not work on those systems?

**Reply by xoxys** [reply]
I can also move this to another category if thats  the problem. The intention was obviously to report an issue but as you prefere to discuss it first I used the discussion instead of a real Issue.

**Reply by viceice** [reply]
- https://github.com/renovatebot/renovate/issues/9987

**Reply by xoxys** [reply]
Thanks. As this issue is open since a few months is there maybe a workaround to get it working on a short term?

**Reply by viceice** [reply]
I don't think so

**Reply by xoxys** [reply]
Don't really understand the down vote on https://github.com/renovatebot/renovate/discussions/13422#discussioncomment-1924363 but ok :shrug:
